stix2/environment.py

```diff
@@ -189,16 +189,31 @@ def creator_of(self, obj):
             return None
 
     @staticmethod
-    def object_similarity(obj1, obj2, prop_scores={}, **weight_dict):
+    def object_similarity(
+        obj1, obj2, prop_scores={}, ds1=None, ds2=None,
+        ignore_spec_version=False, versioning_checks=False,
+        max_depth=1, **weight_dict
+    ):
         """This method returns a measure of how similar the two objects are.
 
         Args:
             obj1: A stix2 object instance
             obj2: A stix2 object instance
             prop_scores: A dictionary that can hold individual property scores,
                 weights, contributing score, matching score and sum of weights.
-            weight_dict: A dictionary that can be used to override settings
-                in the similarity process
+            ds1: A DataStore object instance representing your graph
+            ds2: A DataStore object instance representing your graph
```

Comment:
I think these two parameters could be explained better, and noted as optional. Maybe:

```suggestion
            ds1 (optional): A DataStore object instance from which to pull related objects
            ds2 (optional): A DataStore object instance from which to pull related objects
```

We could also add a note to the docstring (outside the Args section) about what will happen when ds1, ds2 are provided vs. when they won't.

Any changes to this docstring should then also be propogated to the other docstrings for consistency.